en/docs/develop/user-information-recovery-service.md

```diff
@@ -1,5 +1,11 @@
 # User Information Recovery
 
+!!! warning
+
+    The functionalities provided by this SOAP API are provided through the 
```

Comment:
Shall we use this text? It's shorted an more concise.

```
The functionalities enabled by this SOAP API are available with the [Account Recovery REST APIs](../../using-the-account-recovery-rest-apis/) with better performance. Note that it is recommended to use REST APIs wherever possible.
```

Could we also check if this is only applicable to 5.10?

Reply:
Fixed. This applies to all recent versions of IS. I've sent separate PRs for other versions.